guides/release/components/template-tag-format.md

```diff
@@ -0,0 +1,285 @@
+The template tag format is a powerful, new way to write components in Ember. It's a single-file format that combines the component's JavaScript and Glimmer template code. The `<template>` tag is used to keep a clear separation between the template language and the JavaScript around it.
+
+Template tag components use the file extension `.gjs`. This abbreviation is short for "Glimmer JavaScript". The file extension `.gts` is also supported for TypeScript components.
+
+This new format is the official future of Ember's component authoring story, and is stable and usable today. We expect it to become the recommended way of authoring all Ember apps in the near future, once we are satisfied that we have sufficiently polished up all the corners of the implementation.
+
+## Writing template tag components
+
+Just like with separate JavaScript and Glimmer template files, the template tag format has the concept of template-only components and class-based components. Let's take a closer look at how these concepts compare between both component formats in the next section.
+
+### Template-only components
+
+The following template-only component was created in a previous section to extract an avatar layout into a reusable component.
+
+```handlebars {data-filename="app/components/avatar.hbs"}
+<aside>
+  <div class="avatar" title={{@title}}>{{@initial}}</div>
+</aside>
+```
+
+This layout can be turned into a template tag component by wrapping the code in a `<template>` tag and changing the file extension to `.gjs`.
+
+```text {data-filename="app/components/avatar.gjs"}
```

Comment:
```suggestion
```gjs {data-filename="app/components/avatar.gjs"}
```
Perhaps?

Though, last i tried this, showdown-prism didn't support gjs in prerender/fastboot mode.

Reply:
This crashes the app at runtime:

```
Uncaught (in promise) Error: Assertion Failed: Language "gjs" not found. Have you configured Prism correctly?
```

So it would require changes in https://github.com/ember-learn/guidemaker-ember-template. Seems okay to leave as is until highlighting is properly supported?

Reply:
Ya